resources/translations.mjs

```diff
@@ -101,8 +101,420 @@ export const todos = {
+        "Pollo de goma lujoso",
+        "Pescado suave práctico",
+        "Camisa de bronce ergonómica",
+        "Tocino de plástico artesanal",
+        "Pantalones de plástico sin marca",
+        "Salchichas de madera modernas",
+        "Zapatos de acero hechos a mano",
+        "Bicicleta de acero rústica",
+        "Hermosa ensalada congelada",
+        "Pollo bronce hecho a mano",
+        "Bicicleta de granito elegante",
+        "Salchichas de concreto genérico",
+        "Atún de plástico increíble",
+        "Queso fresco a medida",
+        "Tocino fresco electrónico",
+        "Coche de madera con licencia",
+        "Pescado reciclado",
+        "Guantes suaves prácticos",
+        "Chips suaves prácticos",
+        "Pollo pequeño y suave",
+        "Ratón fresco inteligente",
+        "Ratón de metal moderno",
+        "Sabrosos guantes de granito",
+        "Camisa de acero pequeña",
+        "Ordenador de concreto refinado",
+        "Camisa elegante congelada",
+        "Zapatos de concreto inteligentes",
+        "Coche de goma hecho a mano",
+        "Toallas de goma elegantes",
+        "Sombrero de hormigón sin marca",
+        "Pescado de plástico increíble",
+        "Increíbles zapatos frescos",
+        "Hermosa mesa de madera",
+        "Teclado de madera genérico",
+        "Chips de madera reciclados",
+        "Increíbles chips de goma",
+        "Camisa de granito ergonómico",
+        "Sabroso teclado congelado",
+        "Hermoso jabón de acero",
+        "Bicicleta congelada elegante",
+        "Silla de acero reciclada",
+        "Salchichas de bronce modernas",
+        "Elegante queso de madera",
+        "Pequeñas salchichas de plástico",
+        "Zapatos congelados lujosos",
+        "Salchichas de plástico elegantes",
+        "Salchichas frescas artesanales",
+        "Increíble silla suave",
+        "Jabón de madera reciclado",
+        "Pescado reciclado",
+        "Guantes suaves prácticos",
+        "Chips suaves prácticos",
+        "Pollo pequeño y suave",
+        "Ratón fresco inteligente",
+        "Ratón de metal moderno",
+        "Sabrosos guantes de granito",
+        "Camisa de acero pequeña",
+        "Computadora de concreto refinada",
+        "Camisa de goma elegante",
+        "Zapatos de concreto inteligentes",
+        "Hermosa mesa de madera",
+        "Teclado de madera genérico",
+        "Chips de madera reciclados",
+        "Increíbles chips de goma",
+        "Camisa de granito ergonómico",
+        "Sabroso teclado congelado",
+        "Hermoso jabón de acero",
+        "Bicicleta congelada elegante",
+        "Silla de acero reciclada",
+        "Salchichas de bronce modernas",
+        "Elegante queso de madera",
+        "Pequeñas salchichas de plástico",
+        "Zapatos congelados lujosos",
+        "Salchichas de plástico elegantes",
+        "Salchichas frescas artesanales",
+        "Increíble silla suave",
+    ],
+    de: [
```

Comment:
Markus had a look to the german words and this looks good to him, except there's some duplication but maybe that's expected.

Reply:
awesome - I updated the list and removed the duplicate entries for the German translations

Reply:
We need to have at least as much entries as we have todo items to add, right? Or otherwise in the tests we need to use some modulo operation on the array length (this could be the good thing to do anyway...). What do you think?

Reply:
yes exactly!